packages/opencensus-resource-util/src/index.ts

```diff
@@ -13,3 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Optional (may just be my opinion): I tend to like to have `index.ts` just export symbols and then have more clearly named files that do the heavy lifting. E.g. calling this one `detect-resource.ts` or similar and same for its test.

Reply:
Make Sense, Done.